wolfcrypt/src/cryptocb.c

```diff
@@ -194,10 +197,12 @@ WOLFSSL_API void wc_CryptoCb_InfoString(wc_CryptoInfo* info)
         printf("Crypto CB: %s %s (%d)\n", GetAlgoTypeStr(info->algo_type),
             GetHashTypeStr(info->hmac.macType), info->hmac.macType);
     }
+#ifdef WOLF_CRYPTO_CB_CMD
     else if (info->algo_type == WC_ALGO_TYPE_NONE) {
-        printf("Crypto CB: %s %s (%d)\n", GetAlgoTypeStr(info->algo_type),
+        printf("Crypto CB: CMD %s (%d)\n", GetAlgoTypeStr(info->algo_type),
```

Comment:
Too many inputs to the printf.
`./configure --enable-cryptocb CPPFLAGS="-DDEBUG_CRYPTOCB -DWOLF_CRYPTO_CB_CMD" && make`

```
wolfcrypt/src/cryptocb.c: In function ‘wc_CryptoCb_InfoString’:
wolfcrypt/src/cryptocb.c:202:37: error: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘const char *’ [-Werror=format=]
         printf("Crypto CB: CMD %s (%d)\n", GetAlgoTypeStr(info->algo_type),
                                    ~^
                                    %s
             GetCryptoCbCmdTypeStr(info->cmd.type), info->cmd.type);
             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
wolfcrypt/src/cryptocb.c:202:16: error: too many arguments for format [-Werror=format-extra-args]
         printf("Crypto CB: CMD %s (%d)\n", GetAlgoTypeStr(info->algo_type),
                ^~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
Makefile:6705: recipe for target 'wolfcrypt/src/src_libwolfssl_la-cryptocb.lo'
```

Reply:
My bad.  Was toggling between two different thoughts.  This is the first and last time I make code changes via the web API.  Lesson learned.